### PR TITLE
Fix compilation with clang11 on FreeBSD 13.0

### PR DESCRIPTION
**Pull request by karolyi**
This fixes the compilation error mentioned in https://github.com/unbit/uwsgi/issues/2238#issuecomment-819929450.

### CODE REVIEW

**Comment by karolyi** [comment]
@xrmx can you please merge this ASAP? it is blocking a lot of my my uwsgi deployments.

**Comment by karolyi** [comment]
bump

File: plugins/gevent/gevent.c [hunk 1]
```diff
@@ -389,7 +389,7 @@ static void gil_gevent_get() {
 }
 
 static void gil_gevent_release() {
-	PyGILState_Release((PyGILState_STATE) pthread_getspecific(up.upt_gil_key));
+	PyGILState_Release((long) pthread_getspecific(up.upt_gil_key));
```
Comment (xrmx) [inline plugins/gevent/gevent.c:392]: Please keep the proper type `(PyGILState_STATE)(long)`

Comment (karolyi) [reply plugins/gevent/gevent.c:392]: Nice! I'm not a C expert so I wouldn't know.

I have looked into the code and the function `pthread_getspecific` has different casts throughout all the code. Is that intentional? If not, it's just as a security risk as this one, imo.

Comment (xrmx) [reply plugins/gevent/gevent.c]: It returns void * so you are supposed to cast it to whatever you set it before to.

**Comment by xrmx** [comment]
Merged, thanks!

**Pull request closed without merging**